image.yaml

```diff
@@ -1,25 +1,58 @@
-- name: operator-builder
-  version: 1.0.0
-  from: golang:1.14
-  description: Builder Image for the Operator
+- schema_version: 1
+  name: "operator-builder"
+  version: "7.11"
+  from: "registry.redhat.io/ubi8/go-toolset:1.14.12"
+  description: "Golang builder image for the RHPAM Kogito Operator"
 
   modules:
     repositories:
       - path: modules
     install:
-      - name: org.kie.kogito.goModDownload
-      - name: org.kie.kogito.buildOperator
-- name: operator-runtime
-  version: 1.0.0
-  from: registry.access.redhat.com/ubi8/ubi-minimal:latest
-  description: Runtime Image for the Operator
+      - name: org.kie.kogito.gomoddownloader
+      - name: org.kie.kogito.builder
+
+  osbs:
+    configuration:
+      container:
+        image_build_method: imagebuilder
+        platforms:
+          only:
+            - x86_64
+
+- name: "operator-runtime"
+  version: "7.11"
+  from: "registry.redhat.io/ubi8/ubi-minimal:latest"
+  description: "Runtime Image for the RHPAM Kogito Operator"
+
+  labels:
+    - name: maintainer
+      value: bsig-cloud@redhat.com
+    - name: com.redhat.component
+      value: rhpam-7-kogito-rhel8-operator-container
+    - name: io.k8s.description
+      value: Operator for deploying RHPAM Kogito Application
+    - name: io.k8s.display-name
+      value: Red Hat PAM Kogito Operator
+    - name: io.openshift.tags
+      value: rhpam,kogito,operator
 
   artifacts:
-    - name: manager
+    - name: kie-cloud-operator
```

Comment:
Is this name correct? I know it can be anything but do we want to give any reference to our kie-cloud-operator?

Reply:
@spolti I think `rhpam-kogito-operator-manager` should be the name.

Reply:
updated.